easybuild/framework/easyblock.py

```diff
@@ -2158,6 +2158,7 @@ def package_step(self):
     def fix_shebang(self):
         """Fix shebang lines for specified files."""
         for lang in ['perl', 'python']:
+            shebang_regex = re.compile(r'^#![ ]*.*[/ ]%s[0-9.]*.*' % lang)
```

Comment:
should shebangs like `#!/usr/bin/env python3` and `#!/usr/bin/env python3.6` also be fixed?

Reply:
They are, see enhanced tests. The `[0-9.]*` part allows that the `python` is followed by `3` or `3.6`

Reply:
since there is a dot in the `[0-9.]*` I think this can be simplified: `r'^#![ ]*.*[/ ]%s.*' % lang`

Reply:
Yeah, good point, there's no point in using `[0-9.]*.*` indeed, might as well just use `.*`.

Reply:
fixed in a1cea138c